projects/SCION/programs/basic/tests/basic.nix

```
@@ -0,0 +1,28 @@
+{
+  sources,
+  ...
+}:
+
+{
+  name = "scion";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.scion
+          sources.examples.SCION.scion
```

If we check the CI error, we can see that:

```
       error: attribute 'scion' missing
       at /nix/store/jgp4i0nik0h7i2b7sww3y9mkdrac55r8-source/projects/SCION/programs/basic/tests/basic.nix:16:11:
           15|           sources.modules.programs.scion
           16|           sources.examples.SCION.scion
             |           ^
           17|         ];
```           

And indeed if we go to line 39 in `default.nix`, we can see that the example's name is `basic`

```suggestion
          sources.examples.SCION.basic
```